assets/src/edit-story/components/library/panes/text/karma/textPane.karma.js

```diff
@@ -40,6 +40,13 @@ describe('CUJ: Creator can Add and Write Text: Consecutive text presets', () =>
     fixture = new Fixture();
     fixture.setFlags({ enableSmartTextColor: true });
     await fixture.render();
+
+    await fixture.editor.library.textTab.click();
+    // Give some time for the things to be available for the tests.
+    await fixture.events.sleep(300);
```

Comment:
This test is quite flakey for me locally, ~50% success rate, if I put a bigger number here it's more stable, but if it works on the CI with this value - all good 👍🏻

Reply:
Hm, we could probably raise it to be sure.